script/job_generator/generate_job_yaml.py

```diff
@@ -218,6 +218,23 @@ def verify_image(
     return "".join(verified_container_url)
 
 
+@typechecked
+def append_backend_variables(
+    variables: Dict[str, str], job: Dict[str, Tuple[str, str]]
+):
+    """Is searching for enabled back-ends in the job and appends the back-end
+    variable to variables to enable it in the CI job.
```

Comment:
```suggestion
    """Searches for enabled back-ends in the job parameters and appends the back-end
    variable to variables to enable it in the CI job.
```

Does the Python documentation have something similar to Doxygen's `\p` to mark function parameters? I needed to read it a few times before I realized that `variables` was referring to the parameter.